Animal Competition/Java/Game.java

```diff
@@ -1,50 +1,15 @@
 import java.util.Scanner;
 import java.util.ArrayList;
+import java.util.Random;
```

Comment:
Are you importing the right library?  You're using `Math.random` which lives in `java.lang.Math` but youre importing `java.util.Random`.  Also see this article on the difference between the two random number generator libraries: https://stackoverflow.com/questions/738629/math-random-versus-random-nextintint